diff --git a/dist/index.mjs b/dist/index.mjs
index e7ee6225b8a3817445d4b45bffe28c04d957ffc8..00f554c403a79665dbd7419f487687715944e5cd 100644
--- a/dist/index.mjs
+++ b/dist/index.mjs
@@ -1430,7 +1430,7 @@ async function transformMain(code, filename, options, pluginContext, ssr, custom
 				...options.devServer?.config.oxc,
 				lang: "ts",
 				sourcemap: options.sourceMap
-			}, resolvedMap);
+			}, resolvedMap, options.resolvedConfig);
 			resolvedCode = code;
 			resolvedMap = resolvedMap ? map : resolvedMap;
 		} else {
@@ -1694,6 +1694,7 @@ function vuePlugin(rawOptions = {}) {
 		configResolved(config) {
 			options.value = {
 				...options.value,
+				resolvedConfig: config,
 				root: config.root,
 				sourceMap: config.command === "build" ? !!config.build.sourcemap : true,
 				cssDevSourcemap: config.css?.devSourcemap ?? false,
