diff --git a/nuxt.config.ts b/nuxt.config.ts
--- a/nuxt.config.ts
+++ b/nuxt.config.ts
@@ -86,10 +86,10 @@
       baseUrl: process.env.NUXT_BASE_URL,
       paymentPortalUrl: process.env.NUXT_PAYMENT_PORTAL_URL,
       payApiURL: `${process.env.NUXT_PAY_API_URL}${process.env.NUXT_PAY_API_VERSION}`,
-      tokenRefreshInterval: process.env.NUXT_KEYCLOAK_REFRESH_INTERVAL || 30000, // default 30 seconds
-      tokenMinValidity: process.env.NUXT_KEYCLOAK_MIN_TOKEN_VALIDITY || 120000, // default 2 mins
-      sessionIdleTimeout: process.env.NUXT_CONNECT_SESSION_INACTIVITY_TIMEOUT || 1800000, // default 30 mins
-      sessionExpiredModalTimeout: process.env.NUXT_CONNECT_SESSION_MODAL_TIMEOUT || 120000 // default 2 mins
+      tokenRefreshInterval: Number(process.env.NUXT_KEYCLOAK_REFRESH_INTERVAL || 30000), // default 30 seconds
+      tokenMinValidity: Number(process.env.NUXT_KEYCLOAK_MIN_TOKEN_VALIDITY || 120000), // default 2 mins
+      sessionIdleTimeout: Number(process.env.NUXT_CONNECT_SESSION_INACTIVITY_TIMEOUT || 1800000), // default 30 mins
+      sessionExpiredModalTimeout: Number(process.env.NUXT_CONNECT_SESSION_MODAL_TIMEOUT || 120000) // default 2 mins
     }
   },
 
diff --git a/app/components/Connect/Modal/Session/Expiring.vue b/app/components/Connect/Modal/Session/Expiring.vue
--- a/app/components/Connect/Modal/Session/Expiring.vue
+++ b/app/components/Connect/Modal/Session/Expiring.vue
@@ -9,7 +9,7 @@
 }>()
 
 const timeRemaining = ref(toValue((modalTimeout as number) / 1000))
-setInterval(() => {
+const countdownInterval = setInterval(() => {
   const value = timeRemaining.value - 1
   timeRemaining.value = value < 0 ? 0 : value
 }, 1000)
@@ -42,6 +42,7 @@
 })
 onUnmounted(() => {
   // cleanup
+  clearInterval(countdownInterval)
   window.removeEventListener('keydown', closeModal)
 })
 </script>
